coralproject/talk
# [CORL-227] Add user status actions to the user history drawer

## What does this PR do?

Adds a drop down to ban and suspend a user within the user drawer.

## How do I test this PR?

- Head to the admin section and log in with an admin account
- View a user drawer by selecting a username in either a moderation queue or community table listing
- User drawer should pop open with a `Status` label and action field
- Should be able to ban and suspend users
